admin_guide/metrics-sizing-guidelines.adoc

```diff
@@ -0,0 +1,164 @@
+= Openshift Metrics Sizing Guidelines
```

Comment:
done, for rest will wait on more feedback and then do one big rebase instead of many small